lib/condorExe.py

```diff
@@ -13,36 +12,52 @@
 
 
 class CondorExeError(RuntimeError):
-    """Base class for condorExe module errors"""
+    """Base class for condorExe module errors."""
```

Comment:
Need to mention inheritance of `RuntimeError` class?

Reply:
no, in class signature